app/views/reporting/user_reports/index.html.erb

```diff
@@ -15,5 +15,10 @@
         <%= govuk_link_to label_text(report_type), reporting_user_report_path(report_type), { no_visited_state: true } %>
       </h3>
     <% end %>
+
+    <% report = Reporting::MonthlyReport.latest(report_type: 'caseworker_report') %>
```

Comment:
Could this be moved to the controller?

Reply:
Why have two separate ways off calling the same thing?

`Reporting::MonthlyCaseworkerReport.latest`

and

`report = Reporting::MonthlyReport.latest(report_type: 'caseworker_report')`

If they aren't doing the same thing then I find this API a little confusing without looking at the implementation

Reply:
Good spot. That is a relic!